core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java

```diff
@@ -227,6 +227,11 @@
    */
   private boolean locked;
 
+  /**
+   * Whether rels with Convention.NONE has infinite cost.
+   */
+  private boolean noneConventionHaveInfiniteCost = true;
```

Comment:
Grammatically speaking, this should be `noneConventionHasInfiniteCost` (same with other occurrences)

Reply:
Done. Thanks for the catch!